scripts/superdesk/upload/image-crop-directive.js

```diff
@@ -279,136 +278,132 @@
-                        left: (scope.point.x * img.width) - 15,
-                        top: (scope.point.y * img.height) - 15
+                    var topOffset = (poi.y * img.height) - circleRadius;
+                    var leftOffset = (poi.x * img.width) - circleRadius;
+                    var verticalLeftOffset = leftOffset + circleRadius - (lineThickness / 2);
+                    var horizontalTopffset = topOffset + circleRadius - (lineThickness / 2);
+                    element.find('.image-point__poi').css({
+                        width: img.width,
+                        height: img.height
+                    });
+                    element.find('.image-point__poi__cursor').css({
+                        left: leftOffset,
+                        top: topOffset
                     });
-                    crossLeft.css({
-                        width: (scope.point.x * img.width) - 15,
-                        top: (scope.point.y * img.height)
+                    element.find('.image-point__poi__cross-left').css({
+                        width: leftOffset,
+                        top: horizontalTopffset
                     });
-                    crossRight.css({
-                        width: ((1 - scope.point.x) * img.width) - 19,
-                        top: (scope.point.y * img.height),
-                        left: (scope.point.x * img.width) + 19
+                    element.find('.image-point__poi__cross-right').css({
+                        width: img.width - (leftOffset + (2 * circleRadius)),
+                        top: horizontalTopffset,
+                        left: leftOffset +  (2 * circleRadius)
                     });
-                    crossBottom.css({
-                        height: ((1 - scope.point.y) * img.height) - 19,
-                        top: (scope.point.y * img.height) + 19,
-                        left: (scope.point.x * img.width) + 1
+                    element.find('.image-point__poi__cross-top').css({
+                        height: topOffset,
+                        left: verticalLeftOffset
                     });
-                    crossTop.css({
-                        height: (scope.point.y * img.height) - 15,
-                        left: (scope.point.x * img.width) + 1
+                    element.find('.image-point__poi__cross-bottom').css({
+                        height: img.height - (topOffset + (2 * circleRadius)),
+                        left: verticalLeftOffset,
+                        top: topOffset + (2 * circleRadius)
                     });
                 }
+                // init directive element style
+                element.css({
+                    position: 'relative',
+                    display: 'block'
+                });
+                // load the image in order to know the size
+                var img = new Image();
+                img.onload = function() {
+                    function drawPointsFromModel() {
```

Comment:
there's a circular reference here since the DOM function will reference img. I'm not sure if V8 can garbage collect this.